lib/features/auth/core/constants.dart

```diff
@@ -12,7 +12,7 @@ class UserConfigConstants {
   static String isPINPrintLoginEnabled = "is_pin_print_log_enabled";
   static String isPINLoginEnabled = "is_pin_log_enabled";
   static String enablePINLogin = "enable_pin_login";
-
+  static const String prefKeyVoice = 'pref_key_voice';
   static String isAutoSaveEnabled = "is_auto_save_enabled";
```

Comment:
remove `const` to keep consistency, its not necessary here.

Reply:
oops